src/writer/settings.ts

```diff
@@ -80,6 +81,26 @@ export async function verify(body): Promise<any> {
     return Promise.reject(`max number of strategies is ${strategiesLimit}`);
   }
 
+  try {
```

Comment:
Maybe we should move this check and strategies limit check to a new function similar to `validateSpaceSettings`